ui/lib/etherscan-prefix-for-network.js

```diff
@@ -1,24 +1,22 @@
-export default function etherscanNetworkPrefix (network) {
-  const net = parseInt(network)
-  let prefix
-  switch (net) {
-    case 1: // main net
-      prefix = ''
-      break
-    case 3: // ropsten test net
-      prefix = 'ropsten.'
-      break
-    case 4: // rinkeby test net
-      prefix = 'rinkeby.'
-      break
-    case 42: // kovan test net
-      prefix = 'kovan.'
-      break
-    case 5: // goerli test net
-      prefix = 'goerli.'
-      break
-    default:
-      prefix = ''
+import * as networkEnums from '../../app/scripts/controllers/network/enums'
+
+/**
+ * Gets the etherscan.io URL prefix for a given network ID.
+ *
+ * @param {string} networkId - The network ID to get the prefix for.
+ * @returns {string} - The etherscan.io URL prefix for the given network ID.
```

Comment:
Nit: As I think you taught me, this dash isn't required for the return annotation
```suggestion
 * @returns {string} The etherscan.io URL prefix for the given network ID.
```

Reply:
![A0odMl](https://user-images.githubusercontent.com/4448075/86838178-78d9f000-c065-11ea-99be-950ea4277476.gif)